src/redux/dragons/dragons.js

```diff
@@ -0,0 +1,34 @@
+const URL = 'https://api.spacexdata.com/v3/dragons';
+
+export const displayFetchedDragons = (dragons) => ({
+  type: DISPLAY_DRAGONS, // eslint-disable-line
+  dragons,
+});
+
+const showDragons = (data) => {
+  console.log(data);
+};
+
+export const fetchDragons = () => (dispatch) => {
+  fetch(URL)
+    .then((resolve) => resolve.json())
+    .then((data) => {
+      dispatch(displayFetchedDragons(data));
+      showDragons(data);
+    })
+    .catch((error) => console.log(error));
+};
+
+const initialState = [];
+
+const dragonsReducer = (state = initialState, action = {}) => {
+  switch (action.type) {
+    case DISPLAY_DRAGONS: // eslint-disable-line
+      return [...state, ...action.dragons];
+    default:
+      console.log('Dispatched action has no matching case');
+      return state;
+  }
```

Comment:
#### [OPTIONAL] Hi, it would be nice if you could move the `state` file to the` configureStore.js` file to check what you did in this task and use `console.log () to show that it was a browser console. Just like the Fetch the Rockets mission.
#
![2022-03-16_19-49-00](https://user-images.githubusercontent.com/63915024/158617868-3d968c3c-dba3-4cce-916d-aff67497177f.png)